node/core/provisioner/src/lib.rs

```diff
@@ -235,8 +209,8 @@ impl ProvisioningJob {
 	) {
 		if let Err(err) = send_inherent_data(
 			self.relay_parent,
-			&self.signed_bitfields,
-			&self.backed_candidates,
+			self.signed_bitfields.clone(),
```

Comment:
Is it preferential to explicitly clone vs cloning the reference inside of send_inherent_data?

Reply:
:shrug: there are two pov here: cloning early can sometimes avoid cloning at all (not here), cloning late sometimes clones when there doesn't need to be a clone.

I can revert that part if desired.